src/lib/forms/PipelineForm.svelte

```diff
@@ -72,4 +81,13 @@
   >
     Is this an alternative route and not the default option?
   </Checkbox>
+  {#if shouldDisplayCoveragePolygonQuestion && props.is_coverage_polygon !== undefined}
+    <Checkbox
+      id={"coverage-polygon-" + id}
+      bind:checked={props.is_coverage_polygon}
+    >
+      Is this a polygon to manually denote what the coverage of the scheme/lcwip
```

Comment:
I don't want us to be in the business of figuring out wording here... suggestion:
"Does this polygon show the coverage of the scheme? (All area considered while making the scheme)"

Reply:
I've got no strong preference, happy to change this but wording will become clear when we know better what this is for 😅